lib/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var watcherServerMiddleware = require('./middleware');
+var serveAssetMiddleware = require('./serve-asset-middleware');
+var watcherMiddleware = require('./watcher-middleware');
+
+module.exports = {
+  'watcherServerMiddleware': watcherServerMiddleware,
```

Comment:
Seems reasonable.  Personally I would not include it in `master` but instead create a minor version branch if a patch version became necessary.  Not sure what the maintainer preference here is though.  Seems like a decision for @joliss @rwjblue @stefanpenner